src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/LengthConverter.cs

```diff
@@ -181,59 +174,61 @@ public override bool CanConvertTo(ITypeDescriptorContext typeDescriptorContext,
         // NOTE - This code is called from FontSizeConverter, so changes will affect both.
         static internal double FromString(string s, CultureInfo cultureInfo)
```

Comment:
Could you revert your changes in this method ? I already made other changes in dotnet/wpf#8622 which I think would be slightly faster and a bit cleaner (I centralized other similar pixel unit).

Reply:
@ThomasGoulet73 Those look like great changes, I'll go through them. Totally missed this PR, sorry.

Reply:
@h3xds1nz, have you reverted the changes as mentioned above ? We are reviewing this PR for the next CTP that we will start.

Reply:
@dipeshmsft I'll fix it within 30m.

Reply:
@dipeshmsft Since the other for changes LengthConverter were included in #9363 instead, I propose I put this PR into draft state right now and I'll do the additional changes I have planned once Thomas's PR is merged, it will be easier.

Reply:
> I propose I put this PR into draft state right now and I'll do the additional changes I have planned once Thomas' PR is merged, it will be easier.

Sure, that works for us. Infact, for us as well this was the initial plan, get Thomas's PR merged and then take a look at this PR once that is done. I will ping once the current CTP is done.